ci/praktika/result.py

```diff
@@ -481,7 +494,8 @@ def from_commands_run(
 
         print(f"> Start execution for [{name}]")
         res = True  # Track success/failure status
-        error_infos = []
+        info_lines = []
+        MAX_LINES_IN_INFO = 300
```

Comment:
This is only for summary of the job, correct? I still can see complete logs/gdb logs?

Reply:
Indeed, can it be the reason for https://github.com/ClickHouse/ClickHouse/issues/84140#issuecomment-3098218635 ?